src/style.scss

```diff
@@ -1,5 +1,12 @@
 @import url(//fonts.googleapis.com/css?family=Open+Sans);
 
+@mixin lineLimit($line) {
+  display: -webkit-box;
+  -webkit-line-clamp: $line;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+}
```

Comment:
`-webkit` might only work on Chrome and Safari. You need to consider IE Edge and Firefox as well. 

Also, maybe you should use postcss to avoid the prefixes. 
https://github.com/postcss/autoprefixer#webpack

Reply:
I've checked it and test on each browser.
![image](https://user-images.githubusercontent.com/13488449/61930894-861bdd00-afc2-11e9-81b5-b081c176d142.png)

Reply:
Interesting. I didn't know that Gecko accpets any other vendor prefix except `-moz`.

Reply:
Only work on **latest** Firefox. 
Firefox support part of -webkit prefix.
https://stackoverflow.com/questions/20594995/why-webkit-line-clamping-does-not-work-in-firefox